website/src/contribute/release-guide.md

```diff
@@ -428,6 +422,44 @@ Javadoc to the Javadoc for other modules that Beam depends on.
   the version number has changed, download a new version of the corresponding
   `<module>-docs/package-list` file.
 
+### Triage release-blocking issues in JIRA
+
+There could be outstanding release-blocking issues, which should be triaged before proceeding to build a release candidate. We track them by assigning a specific `Fix version` field even before the issue resolved.
+
+The list of release-blocking issues is available at the [version status page](https://issues.apache.org/jira/browse/BEAM/?selectedTab=com.atlassian.jira.jira-projects-plugin:versions-panel). Triage each unresolved issue with one of the following resolutions:
+
+For all JIRA issues:
+
+* If the issue has been resolved and JIRA was not updated, resolve it accordingly.
+
+For JIRA issues with type "Bug" or labeled "flaky":
+
+* If the issue is a known continuously failing test, it is not acceptable to defer this until the next release. Please work with the Beam community to resolve the issue.
+* If the issue is a known flaky test, make an attempt to fix the issue. However, if the issue may take too long to fix (to the discretion of the release manager):
```

Comment:
This might not be super clear - I think the release manager should make an attempt to _get the test fixed_ but often by delegating / asking someone else, just like the next bullet.

Reply:
Updated to be more clear that the release manager should try to delegate a fix first then try delegating manual testing.